src/hotspot/share/memory/virtualspace.hpp

```diff
@@ -173,6 +177,7 @@ class VirtualSpace {
   // Need to know if commit should be executable.
   bool   _executable;
 
+  MEMFLAGS _nmt_flag;
```

Comment:
Don't place the variable against the comment below.
```suggestion
  MEMFLAGS _nmt_flag;

```

Reply:
Done.